Lint-Whoppers/nutshell
# logout button

# Description

Logout button works and redirects you to Welcome Page.

Fixes https://github.com/Lint-Whoppers/nutshell/issues/93

## Type of change


# Testing Instructions

1) `git fetch --all`
2) `git checkout sp-login`
3)  Check to see that you can logout and will be redirected to Welcome page, where you can then log in again



# Checklist:
